src/Microsoft.Identity.Web.Certificate/DefaultCredentialsLoader.cs

```diff
@@ -116,6 +117,31 @@ public async Task LoadCredentialsIfNeededAsync(CredentialDescription credentialD
             return null;
         }
 
+        /// <inheritdoc/>
+        public async Task<SigningCredentials?> LoadSigningCredentialsAsync(
+        CredentialDescription credentialDescription,
+        CredentialSourceLoaderParameters? parameters = null)
+        {
+            _ = Throws.IfNull(credentialDescription);
+
+            try
+            {
+                await LoadCredentialsIfNeededAsync(credentialDescription, parameters);
+
+                if (credentialDescription.Certificate != null)
+                {
+                    return new X509SigningCredentials(credentialDescription.Certificate, credentialDescription.Algorithm);
+                }
+
+                return null;
```

Comment:
Returning null if Certificate is not loaded because a null certificate is a valid state. Open to suggestions.